CMSgov/mdct-mcr
# MDCT-2087 - Extract, Transform, Load (ETL)

## Description
[MDCT-2087](https://qmacbis.atlassian.net/browse/MDCT-2087) - Pipeline-based script to run against the database and optionally update text within the formTemplate (useful for updating hint text and similar content). For updates like adding formFields I would consider doing an update-in-place of the whole formTemplate object to the latest.

### How to test
- Create a deployed branch (or use the [deployed branch here](https://d3o32i7pcsc55c.cloudfront.net/))
- View the report and find text you'd like to change (or find it in `mcpar.json`)
- Update the script following the readme
- Run and see if it works!

Reach out to me if you need help

### Changed Dependencies
None

## Code author checklist

## Reviewer checklist (two different people)
